contributors/design-proposals/propagation.md

```diff
@@ -109,6 +125,12 @@ Opinion against this:
 
 1. (From @jonboulle) May cause cross-runtime compatibility issue.
 
+1. It's not possible to validate a pod + mount propagation. Mount propagation
```

Comment:
oh. I didn't realize that. Does that mean the cool annotation thing only works with local volumes?

That would be a bummer. as the annotation could be useful in other ways, such as tagging a cinder based pv so that the pod it gets attached to only lands on vm's when you have a hybrid openstack/baremetal cluster (I do)